store/src/java/com/zimbra/cs/service/account/Auth.java

```diff
@@ -333,7 +334,14 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
                     }
                 } else {
                     if (password != null || recoveryCode != null) {
-                        prov.authAccount(acct, code, AuthContext.Protocol.soap, authCtxt);
+                        try {
+                            prov.authAccount(acct, code, AuthContext.Protocol.soap, authCtxt);
+                        } catch (AccountServiceException ase) {
+                            if (AccountServiceException.CHANGE_PASSWORD.equals(ase.getCode())) {
```

Comment:
should we print the AccountServiceException if anything occurs other than the Change Password?

Reply:
Not needed